src/components/SideNav/index.jsx

```diff
@@ -34,7 +34,7 @@ const SideNav = (props) => {
       <Link to={{ pathname: `${BASE_URL}/users` }} className="ListItem">Users</Link>
       <div>
         <NavLink to={{ pathname: `${BASE_URL}/accounts` }} className="SubListItem">Accounts</NavLink>
-        <NavLink to={{ pathname: `${BASE_URL}/projects` }} className="SubListItem">Projects</NavLink>
+        <NavLink to={{ pathname: '/projects' }} className="SubListItem">Projects</NavLink>
```

Comment:
Hey @neelxie, projects are under a `Cluster`, so i suggest the path to projects is:
```suggestion
        <NavLink to={{ pathname: `${BASE_URL}/projects` }} className="SubListItem">Projects</NavLink>
```

Reply:
@Mubangizi kindly consider taking pull requests serious and please pay keen attention to detail.